### ISSUE TITLE
Add option to align horizontally

### ISSUE THREAD

**Issue by chhuang** [richardgazdik/sketch-align#3]
This could be useful for very long artboards.

**Comment by Raghav789** [comment]
yes, I would like to have all my 24 points icon in a nice 10*10 arrangement.
please consider this some sort of feature request developer.
